integration/cri-api/pkg/apis/services.go

```diff
@@ -121,6 +121,14 @@ type RuntimeService interface {
 	UpdateRuntimeConfig(runtimeConfig *runtimeapi.RuntimeConfig, opts ...grpc.CallOption) error
 	// Status returns the status of the runtime.
 	Status(opts ...grpc.CallOption) (*runtimeapi.RuntimeStatus, error)
+	// RuntimeConfig returns configuration information of the runtime.
+	// A couple of notes:
```

Comment:
Would you please provide that source about this?

Reply:
from: https://github.com/kubernetes/cri-api/blob/5eda686b5b1446f079d074e83f7cf57963fcdb01/pkg/apis/runtime/v1/api.proto#L135-L142